src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/helper/ReportBuilder.kt

```diff
@@ -103,7 +103,7 @@ fun buildReport(
     )
     (1..generateQuestions).forEach { questionIndex ->
       val historicalQuestion = history.addQuestion(
-        code = "QID-$historyIndex-%012d".format(questionIndex),
+        code = "$historyIndex-%012d".format(questionIndex),
```

Comment:
if you're getting rid of the 0 prefix on the question, perhaps this should be simply
```kotlin
"$historyIndex-$questionIndex"
```

Reply:
yeah, to be honest I didn't really think it mattered as codes now are "just a string, whatever that is" :)

I guess I can change to remove the padding, but AFAICT this is just a testing machinery rather than something that affect real historic questions' codes right?

Reply:
it is just a test but… i dunno, it looks odd now that there are 0s in the middle but not at the front

Reply:
Updated: a54b269a2fdad8ad34107edc9ff04ec3bbd655e6

Reply:
> it is just a test but… i dunno, it looks odd now that there are 0s in the middle but not at the front

It's just a string, it could be anything, like `waffles-waffles-1-00000-23` 😝  (anyway, I changed, so it shouldn't disturb you any longer :D)